config.go

```diff
@@ -66,7 +66,16 @@ type Config struct {
 
 	VirtualMachinesStat []string `toml:"virtual_machines_stat" comment:"default ['hyper-v'], available options 'hyper-v'"`
 
-	HardwareInventory bool `toml:"hardware_inventory" comment:"default true"`
+	HardwareInventory      bool                   `toml:"hardware_inventory" comment:"default true"`
+	CPUUtilisationAnalysis CPUUtilisationAnalysis `toml:"cpu_utilisation_analysis"`
+}
+
+type CPUUtilisationAnalysis struct {
```

Comment:
Did you intentionally put here the struct from my PR?

Reply:
The decision was made later to move the detection logic to `cpuWatcher`. I had already added this feature to `Top`.
Wasn't sure about the other PR and therefor left this here for now.
I guess I can remove this struct then?

Reply:
Ah, got it. Yes, I think so.. I thought that it can be useful to have the analyzer as a separate pkg. So it could be possible attach more pkg like `top` there later